DB/projects.json

```diff
@@ -4710,5 +4710,22 @@
         ]
       }
     ]
+  },
+  {
+    "github_username": "tedante",
+    "Projects": [
+      {
+        "github_url": "https://github.com/justcodebeast/zoom-api-helper",
+        "title": "zoom-api-helper",
+        "description": "Zoom API Helper is a Node.js package that provides a helper for simplifies managing meet.",
+        "tech": ["axios", "typescript"]
```

Comment:
remove  `axios`
```suggestion
        "tech": ["typescript"]
```

Reply:
sure, thank you for reviewing @priyankarpal